invenio_app_ils/patrons/anonymization.py

```diff
@@ -175,14 +197,20 @@ def anonymize_patron_data(patron_pid, force=False):
         except NotFoundError:
             pass
 
-    return dropped, indices
+    return dropped, indices, emails
 
 
 def delete_user_account(patron_pid):
     """Deletes a user account from the database, without updating the index."""
     dropped = 0
 
     with db.session.begin_nested():
+        # delete rows from transaction table
```

Comment:
Lars was suggesting to set a global config `DB_VERSIONING_USER_MODEL = None` instead, see [here](https://github.com/inveniosoftware/invenio-db/blob/master/invenio_db/ext.py#L116-L124). WDYT?